src/Microsoft.DotNet.ProjectJsonMigration/MigrationTrace.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.DotNet.ProjectJsonMigration
+{
+  public class MigrationTrace
+  {
+    public static MigrationTrace Instance { get; set; }
+
+    static MigrationTrace ()
+    {
+      Instance = new MigrationTrace();
+    }
+
+    public string EnableEnvironmentVariable => "DOTNET_MIGRATION_TRACE";
```

Comment:
My thoughts were that a user of the migration library could use this value to enable/disable the trace for debugging purposes, rather than hard coding the string into their program.